docs/content/config/environment.md

```diff
@@ -67,6 +67,19 @@ Note: you probably want to [set `POSTFIX_INET_PROTOCOLS=ipv4`](#postfix_inet_pro
 
 Set the timezone. If this variable is unset, the container runtime will try to detect the time using `/etc/localtime`, which you can alternatively mount into the container. The value of this variable must follow the pattern `AREA/ZONE`, i.e. of you want to use Germany's time zone, use `Europe/Berlin`. You can lookup all available timezones [here](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones#List).
 
+##### ENABLE_RSPAMD
+
+!!! warning "Current State"
+
+    Rspamd-support is under active development. Be aware that breaking changes can happen at any time. Moreover, you will _currently_ need to adjust Postfix's configuration _yourself_ if you want to use Rspamd; you may use [`user-patches.sh`][docs-userpatches].
```

Comment:
A bit more context with the mention of what needs to be manually done here would be useful. Otherwise I'm not sure how useful the suggestion is? What sort of changes do users wanting to try this feature need to make to Postfix?

Reply:
I will provide a better explanation.